src/EnergyPlus/WaterUse.cc

```diff
@@ -278,7 +278,6 @@ namespace WaterUse {
         int IOStatus;            // Used in GetObjectItem
         int NumAlphas;           // Number of Alphas for each GetObjectItem call
         int NumNumbers;          // Number of Numbers for each GetObjectItem call
-        int AlphaNum;
```

Comment:
[src/EnergyPlus/WaterUse.cc:281]:(style),[variableScope],The scope of the variable 'AlphaNum' can be reduced.